config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  resources :users, only: [:index, :show]
+  resources :posts, only: [:index, :show]
```

Comment:
- [ ] It would be more proper to nest these routes since posts technically depend on the users.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

Reply:
Fixed.